.github/chainguard/dd-trace-java.release.sts.yaml

```
@@ -0,0 +1,12 @@
+issuer: https://gitlab.ddbuild.io
+
+# DEBUG: ref should be "master"
+subject: "project_path:DataDog/apm-reliability/dd-trace-java:ref_type:branch:ref:sarahchen6/use-octo-sts"
+
+claim_pattern:
+  project_path: "DataDog/apm-reliability/dd-trace-java"
+  ref_type: "branch"
+  # ref: "master"
+
+permissions:
+  contents: "read,write"
```

write includes read:
```suggestion
  contents: "write"
```